src/pages/index.html

```
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <link href="./style.css" rel="stylesheet" />
+    <script type="module" src="./main.ts"></script>
+  </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)